datadog/resource_datadog_powerpack.go

```diff
@@ -589,6 +590,19 @@ func buildDatadogPowerpack(ctx context.Context, d *schema.ResourceData) (*datado
 
 	groupWidget.Definition = definition
 
+	// Set Live span for all powerpack widgets.
+	if v, ok := d.GetOk("live_span"); ok {
```

Comment:
**question:** Should we handle the case for when it's `!ok` as well?

Reply:
Do you mean the case where `live_span` is not set? It's not a required field so it's fine to skip if it's not set.